content/applications/finance/accounting/bank/feeds/enablebanking.rst

```diff
@@ -0,0 +1,41 @@
+===============================================
+Enable Banking as bank synchronization provider
+===============================================
+
+**Enable Banking** is a third party provider aggregating banking information from bank accounts all
+in one place.
+
+.. image:: enablebanking/enablebanking.png
+   :align: center
+   :alt:   Enable Banking logo
+
+**Enable Banking** offers non-intrusive connectivity to ASPSPs’ official APIs across Europe without
+storing data. Odoo synchronizes directly with banks to get access to all bank transactions and
+automatically import them into your database.
+
+Configuration
+=============
+
+Link bank accounts with Odoo
+----------------------------
+
+In Odoo, all bank transactions are directly synchronized and automatically imported into your
+database.
+
+#. Start synchronization by clicking on :menuselection:`Accounting --> Configuration --> Add a Bank Account`;
+#. Select your bank;
+#. Make sure you give your consent to share your account information with Odoo by clicking
+   :guilabel:`Continue
+   authentication`;
+
+   .. image:: enablebanking/enablebankingauth.png
+      :align: center
+      :alt: Enable Banking authentication page
+#. Finally, you are redirected to your bank's login page.
+
+.. seealso::
+   * `Finnish banks <https://tilisy.enablebanking.com/guides/FI/>`_
+   * `Swedish banks <https://tilisy.enablebanking.com/guides/SE/>`_
+   * :doc:`ponto`
+   * :doc:`saltedge`
```

Comment:
- [rst cheat sheet: bullet lists](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#bulleted-list)
```suggestion
   - `Finnish banks <https://tilisy.enablebanking.com/guides/FI/>`_
   - `Swedish banks <https://tilisy.enablebanking.com/guides/SE/>`_
   - :doc:`ponto`
   - :doc:`saltedge`
```

But also, I don't think it's relevant to add onto and saltedge. Also, I think these links should rather be in the seelso section in the intro rather than in the "link bank accounts" section

And last, where do these links come from and why do you want to list them specifically? This one seems more appropriate https://enablebanking.com/open-banking-apis